code/game/jobs/job/civilian.dm

```diff
@@ -246,7 +246,7 @@
 	economic_modifier = 7
 	access = list(access_lawyer, access_sec_doors, access_maint_tunnels, access_heads)
 	minimal_access = list(access_lawyer, access_sec_doors, access_heads)
-
+	alt_titles = list("Sol Gov Representative","Health & Safety Inspector","Company Representative")

```

Comment:
Please specifically test Health & Safety Inspector, I am unsure if & will fuck up that code or not.